test/integration/base.py

```diff
@@ -28,6 +29,8 @@ def __init__(self):
 
 class DBTIntegrationTest(unittest.TestCase):
 
+    prefix = "test{}".format(random.randint(0, 999))
```

Comment:
sure, i'll use the unix timestamp. more exhilarating to roll the dice, but i think you're right here :)